### ISSUE TITLE
Principe de neutralité

### ISSUE THREAD

**Issue by NicolasD27** [francoismari/elyze#30]
J'ai malheureusement pas pu tester l'app en ayant entendu parler pendant l'annonce de la suppression des données sur mashup. 

En lisant le fichier des candidats et les anecdotes en particulier, j'ai trouvé que celles-ci n'étaient pas toutes flatteuses ou inversement. 

J'en vois pas franchement l'intérêt en plus. C'est un peu sympa mais ça peut influencer les gens surtout que c'est pas les candidats qui les ont choisies. 

Je recommande de les supprimer tout simplement, ça participera à calmer les critiques et les signalements je pense

J'espère que vous reviendrez sur le play store bientôt pour pouvoir tester !

**Comment by RemiF1908** [comment]
Je suis d'accord que ces propositions ne sont pas forcément neutre, pour moi il faudrait juste garder des anecdotes neutres (par exemple, chanteur préféré, plat préféré… C'est certes moins pertinent, mais au moins c'est neutre)
